corehq/apps/es/tests/test_esquery.py

```diff
@@ -204,3 +205,32 @@ def test_sort(self):
             {"timeStart": {"order": "asc"}},
         ]
         self.checkQuery(query.sort('timeStart', reset_sort=False), json_output)
+
+    def test_cleanup_before_run(self):
```

Comment:
Would be cool if the tests here were more along the lines of testing what query is sent through to ES (through a mock) so you didn't have to test the private interface `_clean_before_run`. Just a thought, not blocking